ipalib/x509.py

```diff
@@ -216,6 +217,10 @@ def serial_number(self):
         return self._cert.serial_number
 
     @property
+    def serial_number_bytes(self):
+        return self._serial_number
```

Comment:
Why do you cache serial number instead of calling ``self.__get_der_field('serialNumber')`` here?

Reply:
Because it's how it's done for the rest of `_bytes` properties. The original decision was to cache all the `_bytes` attributes since it's unnecessary couple more steps to get them and the certificate represented by the `IPACertificate` instance should not change once the instance is created (notice there's no method to swap the original cert we got by `load_xxx_x509_certificate()`).